src/p2p.rs

```diff
@@ -285,10 +293,15 @@ impl Connection {
                         NetworkMessage::Verack => {
                             init_send.send(())?; // peer acknowledged our version
                         }
-                        NetworkMessage::Block(block) => blocks_send.send(block)?,
+                        NetworkMessage::Block(_) => panic!("previously remapped"),
                         NetworkMessage::Headers(headers) => headers_send.send(headers)?,
                         NetworkMessage::Alert(_) => (),  // https://bitcoin.org/en/alert/2016-11-01-alert-retirement
                         NetworkMessage::Addr(_) => (),   // unused
+                        NetworkMessage::Unknown { command, payload } => {
+                            if command.as_ref() == "block" {
```

Comment:
Would it worth panicking here as well if `command.as_ref()` isn't "block"? Or at least a warning?

Reply:
Should be addressed in 2513f6fd38e6534b558b82ac46284e0857ab432a.